app/models/enums/EErrorMessages.ts

```diff
@@ -0,0 +1,27 @@
+// Error messages from API
+export enum EErrorMessages {
+  ADMIN_ONLY = 'Accès restreint aux administrateurs.',
+  CLOUDINARY_FAILURE = 'Erreur lors de l\'upload de l\'image.',
+  DUPLICATE_MAIL = 'Cette adresse mail est déjà associée à un compte',
+  DUPLICATE_PSEUDO = 'Ce pseudo est déjà utilisé.',
+  INVALID_FILE = 'Le fichier est invalide.',
+  INVALID_FILE_MIMETYPE = 'Le format de ce fichier n\'est pas pris en charge.',
+  INVALID_FILE_SIZE = 'Le fichier est trop volumineux.',
+  INVALID_PASSWORD = 'Mot de passe incorrect.',
+  INVALID_PASSWORD_FORMAT = 'Le format du mot de passe est invalide.',
+  INVALID_PROPOSAL = 'Ce film a déjà été proposé.',
+  EXPIRED_ACCESS_TOKEN = 'Token d\'accès périmé.',
+  EXPIRED_SESSION = 'Session expirée. Veuillez vous reconnecter.',
+  COMPROMISED_SESSION = 'Session compromise. Veuillez vous reconnecter.',
+  INVALID_USER = 'Utilisateur introuvable.',
+  MISSING_PASSWORD = 'Mot de passe requis.',
+  NO_EPISODE_AVAILABLE = 'Aucun créneau disponible.',
+  NOT_FOUND = 'Aucun résultat.',
+  NOT_FOUND_MOVIE = 'Aucun film trouvé',
+  ALREADY_POSTED_MOVIE = 'Le film sélectionné a déjà été proposé.',
+  NOT_FOUND_MOVIE_ID = 'Le film demandé n\'a pas été trouvé dans la base de données.',
+  NOT_FOUND_PROPOSAL = 'Le film demandé n\'a pas été trouvé parmis les proposition en cours.',
+  UNAVAILABLE_EPISODE = 'Ce créneau est déjà réservé.',
+  UNBOOKED_EPISODE = 'Ce créneau n\'est pas réservé.',
+  USER_HAS_PROPOSAL = 'Vous avez déjà une proposition en attente. Vous pourrez réserver un nouveau créneau une fois votre proposition publiée.',
+}
```

Comment:
Maybe there is another reason for you to setup this enum here, but the API actually sends you the strings so if you want to use them you shouldn't need to declare them again and only use the message object of fetch response.